tenable/io/v3/base/schema/explore/search.py

```diff
@@ -13,15 +13,17 @@ class SortSchema(Schema):
     Schema for the sorting sub-object
     '''
     property = marshm_fields.Str()
-    order = marshm_fields.Str(validates=v.OneOf(['asc', 'desc']))
+    order = marshm_fields.Str(validate=v.OneOf(['asc', 'desc']))
 
 
 class SearchSchema(Schema):
     '''
     Schema supporting the search request
     '''
-    fields = marshm_fields.List(marshm_fields.Str())
-    filter = marshm_fields.Nested(FilterSchema)
-    limit = marshm_fields.Int()
-    next = marshm_fields.Int()
-    sort = marshm_fields.List(marshm_fields.Nested(SortSchema))
+    fields = marshm_fields.List(marshm_fields.Str(), allow_none=True,
+                                default=None)
```

Comment:
default should be dump_default and should only be specified if a None result is to be returned (instead of omission)

https://marshmallow.readthedocs.io/en/stable/marshmallow.fields.html#marshmallow.fields.Field